packages/api/app/distribution-routes.js

```diff
@@ -101,7 +100,6 @@ async function ensureAuthorizedOrRedirect(req, res, next) {
 }
 
 router.get('/redirect', handleRedirectRequest);
-router.get('/s3credentials', ensureAuthorizedOrRedirect, handleCredentialRequest);
```

Comment:
The `s3Credentials` code in the `api` package is no longer used and has been removed.

Reply:
Nice catch